Source/BugsnagSessionTrackingApiClient.m

```diff
@@ -6,15 +6,59 @@
 #import "BugsnagSessionTrackingApiClient.h"
 #import "BugsnagConfiguration.h"
 #import "BugsnagSessionTrackingPayload.h"
+#import "BugsnagSessionFileStore.h"
 #import "BugsnagLogger.h"
-#import "Bugsnag.h"
-#import "BugsnagKeys.h"
 #import "BugsnagSession.h"
+#import "BSG_RFC3339DateTool.h"
+
 
 @implementation BugsnagSessionTrackingApiClient
 
 - (NSOperation *)deliveryOperation {
     return [NSOperation new];
 }
 
+- (void)deliverSessionsInStore:(BugsnagSessionFileStore *)store {
+    [self.sendQueue addOperationWithBlock:^{
+        if (!self.config.apiKey)
+            return;
```

Comment:
Is it worth logging a warning if this is the case? I believe this state shouldn't really be allowed.

Reply:
Logging is a good idea. It shouldn’t happen but this is a guard to prevent either crashing or sending an invalid payload if it hasn’t been set.